package.json

```diff
@@ -218,6 +218,9 @@
     "yargs": "^16.1.1",
     "yargs-parser": "^20.2.4"
   },
+  "resolutions": {
+    "puppeteer/**/devtools-protocol": "0.0.939404"
+  },
```

Comment:
ref: https://github.com/GoogleChrome/lighthouse/pull/12534#issuecomment-846241628

rather add this than upgrade puppeteer in this PR